test/scan_test.go

```diff
@@ -24,7 +24,7 @@ func TestSubScan(t *testing.T) {
 	}
 
 	log.Println("Generating files...")
-	err = generateFiles("s1", 10, 10, "../LICENSE")
```

Comment:
Removing generateFiles here makes the following sync step silly, and the "generating files" print a lie. I vote to keep it, as some sort of exercise to become in sync before the real test. Otherwise we might as well skip the mkdir and sync step too.